gcs/src/gcs_sm.cpp

```diff
@@ -137,9 +137,54 @@ gcs_sm_destroy (gcs_sm_t* sm)
 {
     gu_mutex_destroy(&sm->lock);
     gu_cond_destroy(&sm->cond);
+    delete sm->pauses_window;
     gu_free (sm);
 }
 
+double
+gcs_sm_paused_in_window_get (gcs_sm_t* const sm, double window_sec,
```

Comment:
`window_sec` is a rather constant value (changes only if the user changes `gcs.fc_auto_evict_window`), so maybe it would be better to avoid this multiplication every time?